### ISSUE TITLE
 Screen flashes when using fade-in

### ISSUE THREAD

**Issue by martinstark** [jirutka/swaylock-effects#25]
First created this [issue](https://github.com/mortie/swaylock-effects/issues/99) on the mortie repo without realizing it was the abandoned one.

When I enable --fade-in the screen flashes before showing the indicator and then fading in the screenshot background. With or without other effects enabled.

**Comment by jirutka** [comment]
I’ve reproduced it on Sway 1.8. It’s really annoying, but I don’t know how to fix it. :/

**Comment by MithicSpirit** [comment]
From my brief testing, it seems like using `ext-session-lock-v1` causes the screen to go black for a moment until it is drawn to. Perhaps the solution would be to create the window and draw to it before locking the session, but I'm not sure whether that's even possible.

This issue also seems to be present on regular https://github.com/swaywm/swaylock, so maybe this should be reported upstream and use whatever fix they implement (if any).

**Comment by Timarrr** [comment]
as of right now, Archlinux's swaylock doesn't have any discernible flashing, but swaylock-effects does

**Comment by fabien-michel** [comment]
mortie's fork is not affected by this black flash. I've seen it only after swaylock-effects AUR repo switched to jirutka's one.

**Comment by luispabon** [comment]
I don't believe it happens on mortie's fork because it doesn't use `ext-session-lock-v1`
